Makefile

```diff
@@ -189,8 +193,8 @@ default-vmlinux: $(DEFAULT_VMLINUX_NAME)
 .PHONY: install-default-vmlinux
 install-default-vmlinux: $(DEFAULT_VMLINUX_INSTALLPATH)
 
-.PHONY: install-default-rootfs
-install-default-rootfs: $(DEFAULT_ROOTFS_INSTALLPATH)
+.PHONY: install-rootfs
```

Comment:
I'd prefer to leave `install-default-rootfs` as a separate target.  A user installing firecracker-containerd to run containers (and not develop firecracker-containerd) won't need the slow-reboot version.

Reply:
That makes sense. We use this one for "make install"...